src/__tests__/index.js

```diff
@@ -404,3 +405,29 @@ test('can accept classNames instead of style objects', () => {
   )
   expect(render(<Comp />)).toMatchSnapshotWithGlamor()
 })
+
+test('should accept user defined contextTypes', () => {
+  const Child = glamorous.div((props, context) => ({
+    fontSize: context.fontSize,
+  }))
+  Child.contextTypes = {
+    fontSize: PropTypes.number,
+  }
+
+  class Parent extends React.Component {
+    getChildContext() {
+      return {
+        fontSize: 24,
+      }
+    }
+    render() {
+      return <Child />
+    }
+  }
+
+  Parent.childContextTypes = {
+    fontSize: PropTypes.number,
+  }
+
+  expect(render(<Parent />)).toMatchSnapshotWithGlamor()
```

Comment:
Thanks to [`enzyme`](https://github.com/airbnb/enzyme/blob/master/docs/api/render.md) we can actually provide the context without having a `parent`. So this should be able to work as:

```jsx
expect(render(<Childe />, {context: {fontSize: 24}}).toMatchSnapshotWithGlamor()
```